embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/DataSource.kt

```diff
@@ -36,6 +36,11 @@ internal interface DataSource<T> {
      * You should NOT attempt to track state within the [DataSource] with a boolean flag.
      */
     fun disableDataCapture()
+
+    /**
+     * Resets any data capture limits since the last time [enableDataCapture] was called.
+     */
+    fun resetDataCaptureLimits()
```

Comment:
- When are these limits being set? 
- Are they just `limits` or a broader concept like `attributes`?

Reply:
This is a limit that will be applied to data capture, e.g. custom breadcrumbs has a limit of 100. I'd envisioned it being set when `enableDataCapture` is called.

However, perhaps it's worth me looking at creating a lambda or something that gets the limit? That might allow the changeset to be more automatic than it currently is.

Reply:
I assume this will be called when we transition between sessions? Lets keep an eye on this and if there's anything else that always need to be done besides limit resetting, we can rename this and do that here too. This is good for now though if there's nothing else!